pilot/pkg/model/context.go

```diff
@@ -597,6 +599,17 @@ const (
 	NodeMetadataPolicyCheckMaxRetryWaitTime = "policy.istio.io/checkMaxRetryWaitTime"
 )
 
+var (
+	_ = annotations.Register(NodeMetadataPolicyCheck,
+		"Dtermines the policy for behavior when unable to connect to Mixer. If not set, FAIL_CLOSE is set, rejecting requests.")
```

Comment:
Determines typo

Reply:
Also, missing NodeMetadataHTTP10. Seems reasonable to leave out though, as it doesnt take the shape of a normal annotation. Right now its just a feature flag that will very likely go away in 1.2/later anyways